ConsoleApp/Walls.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+
+namespace Snake
+{
+    class Walls
+    {
+        List<Figure> wallList;
```

Comment:
code convention violated here
```suggestion
        private List<Figure> _wallList;
```